### PR TITLE
alias ci/ci:list

### PR DESCRIPTION
**Pull request by KarateCowboy**
use friendly v5 style, sans default flags

fixes https://github.com/heroku/heroku-ci/issues/30

### CODE REVIEW

File: commands/ci/index.js [hunk 1]
```diff
@@ -28,3 +27,8 @@ module.exports = {
   ],
   run: cli.command(co.wrap(run))
 }
+
+module.exports = [
+  cmd,
+  Object.assign({ command: 'list' }, cmd)
```
Comment (dickeyxxx) [inline commands/ci/index.js]: can we also have `hidden: true`?

Comment (KarateCowboy) [inline commands/ci/index.js]: umm, so just add it to the hash?

Comment (KarateCowboy) [inline commands/ci/index.js]: done

File: commands/ci/index.js [hunk 2]
```diff
@@ -9,9 +9,8 @@ function* run (context, heroku) {
   return yield RenderTestRuns.render(coupling.pipeline, { heroku, watch: context.flags.watch })
 }
 
-module.exports = {
+const cmd = {
   topic: 'ci',
-  command: 'list',
   default: true,
```
Comment (dickeyxxx) [inline commands/ci/index.js]: this needs to be removed

**Comment by appleton** [comment]
Thanks! Feel free to merge & ship this to npm whenever you're ready.

**Pull request merged**